osbs/api.py

```diff
@@ -1024,3 +1025,13 @@ def delete_config_map(self, name):
         """
         response = self.os.delete_config_map(name)
         return response
+
+    @contextmanager
+    def retries_disabled(self):
+        """
+        Context manager to disable retries on requests
+        :returns: OSBS object
+        """
+        self.os.retries_enabled = False
+        yield self
```

Comment:
That makes sense, it also makes the code look cleaner. Since `Openshift` object is a singleton there, there is no point in making an additional object to be used in contextmanager.

@twaugh thoughts?